src/EditorFeatures/Test/Utilities/AsyncLazyTests.cs

```diff
@@ -1,104 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-#nullable disable
-
-using System;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Test.Utilities;
-using Roslyn.Test.Utilities;
-using Roslyn.Utilities;
-using Xunit;
-
-namespace Microsoft.CodeAnalysis.UnitTests
-{
-    [Trait(Traits.Feature, Traits.Features.AsyncLazy)]
-    public class AsyncLazyTests
-    {
-        [Fact]
```

Comment:
were these moved?

Reply:
Yes; for some reason we had two files with tests, one in EditorFeatures that wasn't doing anything special there. So moved.